presto-docs/src/main/sphinx/admin/graceful-shutdown.rst

```diff
@@ -0,0 +1,34 @@
+=================
+Graceful Shutdown
+=================
+
+Presto has a graceful shutdown API that can be used exclusively on workers in
+order to ensure that they terminate without affecting running queries, given a sufficient
+grace period.
+
+You can invoke the API like this:
+
+.. code-block:: bash
+
+    curl -v -X PUT -d '"SHUTTING_DOWN"' -H "Content-type: application/json" \
+        http://worker:8081/v1/info/state
+
+You should keep in mind:
+
+* If your cluster is secure, you will need to provide a basic-authorization header, or
```

Comment:
I think it's also worth adding that if [System Information Rules](https://prestosql.io/docs/345/security/file-system-access-control.html#system-information-rules) are configured then the user in HTTP request must have read+write permissions in the system information rules.

Reply:
Done.